Machine_Learning/Unsupervised_Machine_Learning/ML_3_6_Contrast_between_k- means,k-medoids_and_Hierarchical_Clustering(A).md

```diff
@@ -0,0 +1,21 @@
+# **Contrast between k- means, k- medoids and Hierarchical Clustering**
+
+## **Introduction**
+Clustering is a type of unsupervised learning method of machine learning. In the unsupervised learning method, the inferences are drawn from the data sets which do not contain labelled output variable. 
+
+## **Audio Content**
+
+In this audio the given topics are covered:
+* What is Clustering?
+* K- means Clustering
+* K-medoid Clustering
+* Hierarchical Clustering
+* K-means Vs K-medoid Clustering
+* K-means Vs Hierarchical Clustering
+* Plotting and Visualization code snippets of these 3 clustering techniques
+
+## **Audio Link:Click on the image**
+
+[![ML 3.6](https://github.com/tavneetgill/winter-of-contributing/blob/Machine_Learning/Machine_Learning/Unsupervised_Machine_Learning/Assets/Clustering_types.png)](https://drive.google.com/drive/folders/14SxEy7aEPYVqQpGSpH_J3NsQ9HiEzmIK?usp=sharing)

```

Comment:
Change the drive link to this: 'https://drive.google.com/file/d/1RXHSrt8Wlr9KmMz-q38IYkYHHuxbu1Ni/view?usp=sharing' as we just want your audio file.

Reply:
@Isha307 done with the changes